Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/RapidAntigenCoronaTestExtensions.kt

```diff
@@ -30,7 +31,6 @@ fun RACoronaTest?.toSubmissionState(nowUTC: Instant = Instant.now()) = when {
         NEGATIVE -> TestNegative(testRegisteredAt = registeredAt)
         REDEEMED -> TestInvalid
         PENDING -> TestPending
-        // TODO: Should be updated once the logic for OUTDATED tests is in
-        OUTDATED -> TestNegative(testRegisteredAt = registeredAt)
+        OUTDATED -> TestOutdated
```

Comment:
We don't show a date in this state?

Reply:
I did not create the outdated card, but from looking at the initialisation code i cant see any place where a time is expected